controller/Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN curl -sSL -o /usr/local/bin/etcdctl https://s3-us-west-2.amazonaws.com/opdem
     && chmod +x /usr/local/bin/etcdctl
 
 # install confd
-RUN curl -sSL -o /usr/local/bin/confd https://github.com/kelseyhightower/confd/releases/download/v0.8.0/confd-0.8.0-linux-amd64 \
+ENV CONFD_VERSION 0.9.0
```

Comment:
Also since we don't have signed images, we don't benefit from parallel downloads.

I wouldn't normally squawk about just TCP latency, but you've watched `docker pull` go--there is a startup cost to each layer download that is still noticeable. It's probably more than the network stack. Some of the latency seems to be in `docker` itself still, although I know they've improved that some.